client/lib/plans/constants.js

```diff
@@ -79,10 +79,8 @@ export const plansList = {
 		getProductId: () => 1,
 		getStoreSlug: () => PLAN_FREE,
 		getPathSlug: () => 'beginner',
-		getDescription: () => i18n.translate(
-			'Get a free blog and be on your way to publishing your first post' +
-			' in less than five minutes.'
-		),
+		getDescription: () => i18n.translate( 'Get a free blog and be on your way to publishing your first post in less than five minutes.' ),
+		getTargetedDescription: () => i18n.translate( 'Get a free website and be on your way to publishing your first post in less than five minutes.' ),
```

Comment:
Only change is "blog" to "website" since we can't assume all sites are going to be blogs.